Automattic/jetpack
# Protect: Add `popover` at navigation badge

Include popover for checked and no checked items at Navigation.

#### Changes proposed in this Pull Request:

* Add popover in `Badge`.
* Add icon for `notChecked`.

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `Protect`
* Navigate to `admin` page.
* Make sure response_type is either incomplete or incomplete_green
* Check popover when hovering badge.
* Check different icons and text for not checked items
